### PR TITLE
Kudu based tasks: WIF support 

### PR DESCRIPTION
**Pull request by onetocny**
**Task names**: 

- AzureFunctionAppContainerV1
- AzureWebAppV1
- AzureWebAppContainerV1

**Description**: Reintroduce WIF support to Kudu based tasks that had to be removed by #18202

**Documentation changes required:** N

**Added unit tests:** N

**Attached related issue:** N

**Checklist**:
- [X] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

**Comment by patelchandni** [comment]
Why all the files are moved from the common-lib to /operations folder under AzureFunctionAppContainerV1?

**Comment by onetocny** [comment]
> @onetocny Why all the files are moved from the common-lib to /operations folder under AzureFunctionAppContainerV1?
> 
> AzureFunctionAppContainerV1 task was released this week and I think your changes are not based on the latest. I see a lot of overlap betweeen this PR and master

@patelchandni I hear you. Files that are on `operations` folder are part of `azurermdeploycommon`, however as we are transitioning towards `arm-rest-v2` library we have add them directly as they are not a part of that library. Plan is to move them to some shared library in the near future (probably [this one](https://github.com/microsoft/azure-pipelines-tasks-common-packages/tree/main/common-npm-packages/webdeployment-common)). However, to be more safe we would like to do changes gradually.

**Review by ozanaydinMSFT — APPROVED** [review]

**Review by starkmsu — APPROVED** [review]

File: Tasks/AzureFunctionAppContainerV1/azurefunctiononcontainerprovider.ts [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
-import { AzureAppServiceUtility } from 'azure-pipelines-tasks-azurermdeploycommon/operations/AzureAppServiceUtility';
-import { AzureResourceFilterUtility } from 'azure-pipelines-tasks-azurermdeploycommon/operations/AzureResourceFilterUtility';
 import tl = require('azure-pipelines-task-lib/task');
-import { addReleaseAnnotation } from 'azure-pipelines-tasks-azurermdeploycommon/operations/ReleaseAnnotationUtility';
-import { ContainerBasedDeploymentUtility } from 'azure-pipelines-tasks-azurermdeploycommon/operations/ContainerBasedDeploymentUtility';
+import { AzureAppService } from 'azure-pipelines-tasks-azure-arm-rest-v2/azure-arm-app-service';
+import { AzureRMEndpoint } from 'azure-pipelines-tasks-azure-arm-rest-v2/azure-arm-endpoint';
+import { Resources } from 'azure-pipelines-tasks-azure-arm-rest-v2/azure-arm-resource';
+import { AzureEndpoint } from 'azure-pipelines-tasks-azure-arm-rest-v2/azureModels';
+import { AzureAppServiceUtility } from 'azure-pipelines-tasks-azure-arm-rest-v2/azureAppServiceUtility';
```
[the remark this answers is not in the document]
Comment (patelchandni) [reply Tasks/AzureFunctionAppContainerV1/azurefunctiononcontainerprovider.ts:6]: all covered in 3.223.0

[the remark this answers is not in the document]
Comment (onetocny) [reply Tasks/AzureFunctionAppContainerV1/azurefunctiononcontainerprovider.ts:6]: Just for clarification, it was introduced in arm-rest-v2 3.222.0 in #18269.

**Pull request merged**